--- musl/src/malloc/expand_heap.c.orig	2018-12-27 12:34:27.726324054 +0100
+++ musl/src/malloc/expand_heap.c	2018-12-27 12:34:39.781545080 +0100
@@ -50,11 +50,9 @@ void *__expand_heap(size_t *pn)
 
 	if (!brk) {
 		brk = __syscall(SYS_brk, 0);
-		brk += -brk & PAGE_SIZE-1;
 	}
 
-	if (n < SIZE_MAX-brk && !traverses_stack_p(brk, brk+n)
-	    && __syscall(SYS_brk, brk+n)==brk+n) {
+	if (__syscall(SYS_brk, brk+n)==brk+n) {
 		*pn = n;
 		brk += n;
 		return (void *)(brk-n);
